lua/entities/starfall_processor/shared.lua

```diff
@@ -94,7 +94,7 @@ function ENT:SetupFiles(sfdata)
 	if SERVER then
 		if self.instance and self.instance.ppdata.models and self.instance.mainfile then
 			local model = self.instance.ppdata.models[self.instance.mainfile]
-			if model and self:ValidateCustomModel(model) then
+			if model and SF.CheckModel(model, self.owner) then
```

Comment:
❓ Do we want to be assigning `model` here so we get the benefit of `SF.NormalizePath`?

Reply:
Since we're not sending it into a gamemode hook, I think it's fine. Source engine will resolve any normalization needed on its own.